### ISSUE TITLE
Abnormal knee extension

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5910]
**Preferred term label:** Abnormal knee extension

**Synonyms**

**Definition (free text, please give PubMed ID)** Limited extension or hyperextension of the knee.

**Parent term (use hpo.jax.org/app)** Abnormality of the knee HP:0002815

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
This could be the parent term for: Limited knee extension HP:0003066; Hyperextensibility of the knee HP:0010500

**Comment by pnrobinson** [comment]
I think this is actually a synonym. https://hpo.jax.org/app/browse/term/HP:0003066
This term (https://hpo.jax.org/app/browse/term/HP:0005085) is actually a parent term, it was misplaced. I will fix it. Thanks for pointing out this inconsistency.
